src/client/sandbox/node/index.js

```diff
@@ -101,4 +107,36 @@ export default class NodeSandbox extends SandboxBase {
         this.win.attach(window);
         this.element.attach(window);
     }
+
+    static processSelector (selector) {
+        var attributeSelectorRegEx = /\[([\w-^]+)=(.+?)]/g;
+        var stringifySelector      = function (value) {
+            if (value === null)
+                return 'null';
+
+            else if (value === void 0)
+                return 'undefined';
+
+            return selector;
+        };
+
+        selector = stringifySelector(selector);
+
+        return selector.replace(attributeSelectorRegEx, (str, name, value) => {
+            var isAttributeValueBeginWithSelector = endsWith(name, '^');
+
+            name = name.replace(/\^$/g, '');
+            if (domProcessor.URL_ATTRS.indexOf(name) !== -1) {
+                value = value.replace(/^['"]|['"]$/g, '');
+                value = urlUtils.getProxyUrl(value);
```

Comment:
I don't know scenario than this may throw.
Url value can begins not only with `http://`, `https://`, `//`.
For example:
```javascript
link.setAttribute('href', 'index.html');
document.querySelector('[href=index.html]');
```